File: event-driven-microservices/java/templates/cqrs-eventsourcing-axon/src/main/java/com/rolandsall24/cqrseventsourcingaxon/model/Product.java
package com.rolandsall24.cqrseventsourcingaxon.model;

import lombok.Data;
import lombok.Getter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Data
@Entity
public class Product {

    @GeneratedValue
    @Id
    private int productId;
    private String name;
    private long serialNumber;
}

File: design-patterns/Behavioral Pattern/Mediator-CQRS/dotnet-core/mediator-cqrs/entities/Product.cs
namespace mediator_cqrs.entities;

public class Product
{
    public int ProductId { get; set; }
    public string Name { get; set; }
    public long SerialNumber { get; set; }

    public Product(int productId, string name, long serialNumber)
    {
        ProductId = productId;
        Name = name;
        SerialNumber = serialNumber;
    }

}